packages/web-components/src/components/va-modal/va-modal.tsx

```diff
@@ -443,7 +437,7 @@ export class VaModal {
                       <li class="usa-button-group__item">
                         {!unstyled && (
                           <va-button
-                            onClick={e => this.handlePrimaryButtonClick(e)}
+                            onClick={e => this.handleSecondaryButtonClick(e)}
```

Comment:
Not related to the issue, but fixing bug in USWDS modal where both secondary button was performing the action of the primary button.